code/__DEFINES/components.dm

```diff
@@ -194,6 +203,23 @@
 //NTnet
 #define COMSIG_COMPONENT_NTNET_RECEIVE "ntnet_receive"			//called on an object by its NTNET connection component on receive. (sending_id(number), sending_netname(text), data(datum/netdata))
 
+//Nanites
+#define COMSIG_HAS_NANITES "has_nanites"						//() returns TRUE if nanites are found

```

Comment:
This signal shouldn't exist.

The 2 primary uses of it I see are the disease symptoms and the nanite chamber. Both of these since they are a game construct that interacts directly with the nanite feature are more acceptable as a GetComponent than most things. Idealy it shouldn't use GetComponent but not everything surrounding this sort of interaction has been made compatible with components yet.

The problem at its base is with having a component represent a physical thing that can be interacted with. I'll really need to consider what to do with these sorts of problems in the future.